GDJS/Runtime/runtimeobject.js

```diff
@@ -970,11 +984,11 @@ gdjs.RuntimeObject.prototype.hasBehavior = function(name) {
  * De/activate a behavior of the object.
  *
  * @param name {String} The behavior name.
- * @param enable {boolean} true to activate the behavior
+ * @param status {boolean} Set to true for enabled. False for Disabled.
  */
-gdjs.RuntimeObject.prototype.activateBehavior = function(name, enable) {
+gdjs.RuntimeObject.prototype.activateBehavior = function(name, status) {
```

Comment:
please keep enable, status is too vague. enable is more descriptive for a boolean.